ArduCopter/Copter.cpp

```diff
@@ -490,6 +490,13 @@ void Copter::loop_rate_logging()
 {
     if (should_log(MASK_LOG_ATTITUDE_FAST) && !copter.flightmode->logs_attitude()) {
         Log_Write_Attitude();
+        Log_Write_PIDS(); // only logs if PIDS bitmask is set
+    }
+    if (should_log(MASK_LOG_FTN_FAST)) {
+        AP::ins().write_notch_log_messages();
+    }
```

Comment:
```suggestion
```
This is down on 501 too

Reply:
yep...something in the rebase I guess that I messed up...will remove

Reply:
fixed